website/static/_redirects

```diff
@@ -82,13 +82,13 @@ https://reactnative.dev/docs/0.74/ram-bundles-inline-requires            /docs/o
 /docs/threading-model                   /architecture/threading-model
 /docs/architecture-glossary             /architecture/glossary
 
-/docs/next/architecture-overview             /architecture/overview
-/docs/next/fabric-renderer                   /architecture/fabric-renderer
-/docs/next/render-pipeline                   /architecture/render-pipeline
-/docs/next/xplat-implementation              /architecture/xplat-implementation
-/docs/next/view-flattening                   /architecture/view-flattening
-/docs/next/threading-model                   /architecture/threading-model
-/docs/next/architecture-glossary             /architecture/glossary
```

Comment:
You need to add new redirect: both `/docs/next/xyz` and `/docs/xyz` should redirect to `/architecture/xyz`

Reply:
Do we have any live references, elsewhere on the internet, to these old pages?

Potentially, there's another case we have been missing up to this point. We may want to **add** redirects from e.g. `/docs/0.75/architecture-overview` to `/architecture/overview`.

Reply:
But yeah, let me reduce this PR to debugging links only — will leave this with you.